.ci/pytorch/win-test-helpers/build_pytorch.bat

```diff
@@ -110,6 +118,25 @@ if "%USE_CUDA%"=="1" (
   set CMAKE_CUDA_COMPILER_LAUNCHER=%TMP_DIR%/bin/randomtemp.exe;%TMP_DIR%\bin\sccache.exe
 )
 
+if "%USE_XPU%"=="1" (
+  :: Install oneAPI bundle
+  if exist "C:\Program Files (x86)\Intel\oneAPI\setvars.bat" (
+      echo "oneAPI bundle already installed"
+  ) else (
+      echo "setvars.bat not found, installing oneAPI bundle"
+      call .ci\pytorch\win-test-helpers\installation-helpers\install_oneapi_bundle.bat
+      echo "oneAPI bundle installed"
+  )
+
+  :: Set up conda environment
```

Comment:
Line 29 has `call %INSTALLER_DIR%\activate_miniconda3.bat`, why still need this part? Is there any special needs for xpu env setup?